src/lib.rs

```diff
@@ -441,6 +441,22 @@ impl Rng {
         }
     }
 
+    /// Fill a byte slice with random data.
+    #[inline]
+    pub fn fill(&self, slice: &mut [u8]) {
+        for item in slice {
+            *item = self.u8(..);
```

Comment:
This implementation could be more efficient. There's probably a way to generate a `u64` at a time and serialize that into the byte slice instead of repeatedly generating `u8`'s.

Reply:
Yes, was thinking about that. Let me look into it.

Reply:
Ok, check https://github.com/smol-rs/fastrand/pull/35/commits/5c6b860d9ddf034c4a70d36f1d378030ef5de87d

Much faster. I don't know if there's a faster way without using `unsafe`.

Reply:
Benchmarks:
```
  test fill             ... bench:          29 ns/iter (+/- 5)
  test fill_naive       ... bench:         343 ns/iter (+/- 33)
```